composer.json

```diff
@@ -13,7 +13,7 @@
         "mikey179/vfsStream": "~1.2",
         "phpunit/phpunit": "~4.8",
         "symfony/css-selector": "~2.8",
-        "drush/drush": "8.1.11",
+        "drush/drush": "8.1.12",
```

Comment:
This was a flail while I was debugging drush-ops/drush#2611, but I left it here because hey...can't hurt.